lib/server/auth.ts

```diff
@@ -1,23 +1,31 @@
-import jwt, { Secret } from 'jsonwebtoken';
+import { jwtVerify, JWTPayload } from 'jose';
 
-export function getJwtSecretKey(): Secret {
-    // Secret is hardcoded for now
-    // TODO: Need to figure out where the secret will be stored
-    const secret = Buffer.from('Zn8Q5tyZ/G1MHltc4F/gTkVJMlrbKiZt', 'base64');
+/**
+ * Get the JWT Secret
+ * @returns 
+ */
+export function getJwtSecretKey(): Uint8Array {
+    const secret = process.env.JWT_SECRET
```

Comment:
_Just a side note:_ We should probably set up a mechanism somewhere to easily rotate  secret keys if needed. It's good practice to rotate them every so often to improve site security.

@briri , we should probably create an issue for this so we are reminded of it at a later stage?

Reply:
My thought was to store this and probably other variables we have yet to identify in AWS Secrets manager which you can configure to rotate on a schedule. 

@jupiter007 can we move this `process.env.JWT_SECRET` to its own function for now? That will make it easier to update when we do swap in Secrets Manager.

Reply:
Thanks @andrewebdev and @briri. Using AWS Secrets Manager is a great idea. I'll add a placeholder function for now.

Reply:
I'll also create a ticket to use Secrets Manager in the NextJS app.

Reply:
I created this ticket to implement Secrets Manager on the frontend: https://github.com/CDLUC3/dmsp_frontend_prototype/issues/62

@briri should I create another ticket for setting up AWS Secrets Manager?